docs/AboutUs.md

```diff
@@ -38,15 +38,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
-### Jean Doe
+### Jony
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jonyxzx.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jonyxzx)]
+[[portfolio](team/jonyxzx.md)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Improve find feature
```

Comment:
Good use of the "Responsibilities" section of the DG to specify your responsibilities. +1 cool ponts.